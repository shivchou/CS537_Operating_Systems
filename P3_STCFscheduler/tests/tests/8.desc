Testing give_cpu() with only one process
