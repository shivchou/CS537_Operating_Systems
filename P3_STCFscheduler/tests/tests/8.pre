./edit-makefile.sh ../solution/Makefile test_8 > ../solution/Makefile.test
cp -f tests/test_8.c ../solution/test_8.c
cd ../solution/
make clean
make -f Makefile.test clean
make -f Makefile.test xv6.img
make -f Makefile.test fs.img
cd ../tests
