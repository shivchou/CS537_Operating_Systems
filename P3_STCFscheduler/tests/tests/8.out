XV6_TEST: scheduler : STCF
XV6_TEST: give_cpu() test started
XV6_TEST: before give_cpu
XV6_TEST: after give_cpu
XV6_TEST: give_cpu() test finished
